targets/unity-v2/source/ExampleTestProject/Assets/Testing/Editor/PlayFabPackager.cs

```diff
@@ -15,12 +15,18 @@ class TestAppPostBuildProcessor : IPostprocessBuildWithReport
         public int callbackOrder { get { return 0; } }
         public void OnPostprocessBuild(BuildReport report)
         {
-            OnPostprocessBuildiOS(report);
+            try {
+                OnPostprocessBuildiOS(report);
+            }
+            catch(Exception e)
+            {
+                Debug.Log("Unhandled exception while attempting to add iOS frameworks with: \n" + e.Message);
+            }
         }
 
         private void OnPostprocessBuildiOS(BuildReport report)
         {
-#if UNITY_IOS
+#if UNITY_IOS || UNITY_MAC
```

Comment:
Why add UNITY_MAC here?

Reply:
Paul requested it offline, we have 2 jenkins jobs for mac and iOS that may leverage this codepath.